README.md

```diff
@@ -24,6 +25,7 @@ property correlationId ${nukleus:newCorrelationId()}
 
 accept "nukleus://receiver/streams/sender"
        option nukleus:route ${routeRef}
+       option nukleus:authorization [0x00 0x03 0x80]
```

Comment:
Example should change now that we use `long` literal in hex

Reply:
done